practice-project-api/routes/calculateRoutes.ts

```diff
@@ -0,0 +1,39 @@
+import { Router, Request, Response } from "express";
+
+export const calculateRoute = Router();
+
+calculateRoute.post("/calculate", (req: Request, res: Response, next) => {
+
+  enum calculatorOperation {
+    Multiply = '*',
+    Divide = '/',
+    Add = '+',
+    Subtract = '-'
+  }
+
+  interface Calculate {
+    firstValue: number,
+    secondValue: number,
+    operation: calculatorOperation,
+  };
+
+  const input: Calculate = {
+    firstValue: req.body.firstValue,
+    secondValue: req.body.secondValue,
+    operation: req.body.operation,
+  };
+
+  let calculate = (firstValue: number, operation: calculatorOperation, secondValue: number): number => {
```

Comment:
this doesn't need to be a function and really needs to be the only thing in this post function...